src/assets/app.extensions.json

```diff
@@ -1180,11 +1180,21 @@
           "key": "title",
           "title": "APP.DOCUMENT_LIST.COLUMNS.NAME",
           "type": "text",
-          "class": "adf-ellipsis-cell adf-expand-cell-5",
+          "class": "adf-ellipsis-cell adf-expand-cell-3",
           "sortable": true,
           "template": "app.columns.libraryName",
           "desktopOnly": false
         },
+        {
+          "id": "app.favorite.libraries.description",
+          "key": "description",
+          "title": "APP.DOCUMENT_LIST.COLUMNS.DESCRIPTION",
+          "type": "text",
+          "class": "adf-ellipsis-cell adf-expand-cell-5",
+          "sortable": true,
```

Comment:
are you sure you wan't to make it sortable?

Reply:
why not?

Reply:
it's a description I don't where it could be useful to sort by description

Reply:
it makes no sense as for me, you want to sort the large pieces of text alphabetically?

Reply:
I have no strong opinion on this. It doesn't really make sense sorting descriptions as it contains sentences but since it's possible to do it why not allow it. Perhaps someone will find it useful one day.

Reply:
It's not sortable anymore!

Reply:
I don't have a strong opinion either. I'm happy with both :) Up to you. Thnx

Reply:
it "COULD" contain sentences, but it might as well contain simple words, and it might be useful to sort based on that. Should be customer's decision if he wants to sort that column or not, we shouldn't take away this possibility. 
sortable=false makes sense on the Thumbnails column, but a description should be sortable

Reply:
The customer can always overwrite and customise the columns in the way they need them to be. Am I correct? @davidcanonieto @DenysVuika? This is just a default stuff

Reply:
It is fine. let's leave it sortable